api/src/services/email.service.ts

```diff
@@ -561,15 +561,14 @@ export class EmailService {
   }
 
   public async lotteryReleased(
-    user: User,
     listingInfo: listingInfo,
     emails: string[],
     appUrl: string,
   ) {
     try {
-      const jurisdiction = listingInfo.juris
-        ? await this.getJurisdiction([{ id: listingInfo.juris }])
-        : user.jurisdictions[0];
+      const jurisdiction = await this.getJurisdiction([
```

Comment:
Do you know why this used to be conditional?

Reply:
We looked at this as a team in bloom eng workshop last week and Emily noted it was copy pasta from another function and was not needed for this.